scripts/run_endpoint.sh

```diff
@@ -6,7 +6,7 @@
 if [ -n "$TESTCASE" ]; then
     case "$TESTCASE" in
     # TODO: add supported test cases here
-    "versionnegotiation"|"handshake"|"transfer"|"retry"|"multiconnect"|"keyupdate")
+    "versionnegotiation"|"handshake"|"transfer"|"retry"|"multiconnect"|"keyupdate"|"chacha20")
```

Comment:
For the time being, can we just claim "server" support instead?

Reply:
And just append ChaCha/Poly to the cipher suites?

Reply:
I looked at the QNS code, and it expects the client to drive the chacha-poly negotiation, and it should only send a single cipher suite. If there's multiple, even if one of them is chacha-poly, it will fail the test.

Reply:
Oh, I see what you're asking. I'll do that.